test/unity/toolkits/coreml_export/test_neural_nets_model_exporter.cxx

```diff
@@ -61,7 +61,7 @@ BOOST_AUTO_TEST_CASE(test_object_detector_export_coreml_with_nms) {
     std::shared_ptr<coreml::MLModelWrapper> model_wrapper =
         export_object_detector_model(
             yolo_nn_spec, 13 * 32, 13 * 32, test_class_labels.size(),
-            13 * 13 * 15, std::move(user_defined_metadata),
```

Comment:
I don't understand this change. Why it's added, and why it's restored to its original state again.